src/commands/test/lib/prepare-tests-command.ts

```diff
@@ -41,8 +41,13 @@ export class PrepareTestsCommand extends Command {
     }
   }
 
+  // Override this for things like parameter validation to avoid throwing errors in the constructor
+  protected async runNoClientInner(): Promise<void> {
```

Comment:
I went with a generic name as the method could be used for other things in the future.  I guess it's better having a more specific name and either change it or add other methods if needed.